src/views/postCreation/PostAuthorSelection/PostAuthorSelection.spec.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import { render, fireEvent, cleanup, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { PostAuthorSelection } from './PostAuthorSelection';
+
+const onAuthorTableClick = () => {};
+const authors = [
+  {
+    avatar: 'abc',
+    firstName: 'Max',
+    id: 1,
+    lastAddedPost: {
+      id: 11,
+      title: 'about',
+    },
+    lastName: 'Petrenko',
+    mainDirection: {
+      id: 11,
+      name: 'flue',
+    },
+    mainInstitution: {
+      city: {
+        id: 11,
+        name: 'Kyiv',
+      },
+      id: 11,
+      name: 'Bolnitsa #4',
+    },
+    qualification: 'Therapist',
+  },
+  {
+    avatar: 'abc',
+    firstName: 'Tax',
+    id: 2,
+    lastAddedPost: {
+      id: 11,
+      title: 'about',
+    },
+    lastName: 'Ivanov',
+    mainDirection: {
+      id: 11,
+      name: 'flue',
+    },
+    mainInstitution: {
+      city: {
+        id: 11,
+        name: 'Kyiv',
+      },
+      id: 11,
+      name: 'Bolnitsa #4',
+    },
+    qualification: 'Therapist',
+  },
+  {
+    avatar: 'abc',
+    firstName: 'Pax',
+    id: 3,
+    lastAddedPost: {
+      id: 11,
+      title: 'about',
+    },
+    lastName: 'Avdeev',
+    mainDirection: {
+      id: 11,
+      name: 'flue',
+    },
+    mainInstitution: {
+      city: {
+        id: 11,
+        name: 'Kyiv',
+      },
+      id: 11,
+      name: 'Bolnitsa #4',
+    },
+    qualification: 'Therapist',
+  },
+];
+
+describe('PostAuthorSeclection', () => {
+  afterEach(cleanup);
+
+  it('show authors lastnames recieved from props', () => {
+    const { getAllByTestId } = render(
+      <PostAuthorSelection
+        authors={authors}
+        onAuthorTableClick={onAuthorTableClick}
+        handleOnChange={() => {}}
+      />,
+    );
+    const authorNames = getAllByTestId('item').map((tc) => tc.textContent);
+    const fakeAuthorNames = authors.map((author) => author.lastName);
+    expect(authorNames).toEqual(fakeAuthorNames);
+  });
+
+  it('render table once author existed', async () => {
+    const handleOnChangeMock = jest.fn();
+    render(
+      <PostAuthorSelection
+        authors={authors}
+        searchValue="Ivan"
+        onAuthorTableClick={onAuthorTableClick}
+        handleOnChange={handleOnChangeMock}
+      />,
+    );
+    screen.debug();
+    expect(screen.queryByText(/first name/i));
```

Comment:
nit: I don't think this test tests anything

If you are trying to test presence of `first name` text, you could:
get `getByText` from `render` function and do 
`getByText(/first name/i)` and the test will fail if there are no elements